flatgfa/src/cli/main.rs

```diff
@@ -56,6 +58,15 @@ fn main() -> Result<(), &'static str> {
         }
     }
 
+    if let Some(Command::SeqExport(sub_args)) = args.command {
```

Comment:
Please add a comment here mentioning the "shortcut" for this case, which inputs or outputs raw sequence data instead of GFA files.

Reply:
Still needs a comment.